classes/class-apm-admin-settings.php

```diff
@@ -210,7 +214,13 @@ public static function render_form( $type ) {
 		 * @param string $type tab type.
 		 */
 		public static function render_page_url( $type = '' ) {
-			echo admin_url( '/options-general.php?page=automate-mautic' . $type );
+			$parent = self::get_menu_parent();
+			$admin_url = admin_url( $parent );
+
+			$admin_url = add_query_arg( array(
+			    'page' => 'automate-mautic' . $type,
+			), $admin_url );
+			echo $admin_url;
 		}
 
```

Comment:
Instead of having duplicate code in `render_page_url()` and `get_render_page_url()` why not just call `get_render_page_url()` inside `render_page_url()` and then echo the result?

same as [the_title()](https://developer.wordpress.org/reference/functions/the_title/) does?